src/Chapter2.hs

```diff
@@ -620,7 +625,10 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [Int] -> [Int]
+takeEven (x:y:xs) = x : takeEven xs
```

Comment:
Since the name of variable `y` doesn't matter here, you can use the `_` underscore pattern :slightly_smiling_face: 
```suggestion
takeEven (x:_:xs) = x : takeEven xs
```